### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by raineorshine**
- Add `engines`, `repository`, `homepage`, and `bugs` fields.
- Move `eslint` to `devDependency`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,37 @@
...
+    "url": "https://github.com/e-e-e/book-covers-api/issues"
+  },
   "main": "src/index.js",
   "bin": {
     "bookcovers": "src/cli.js"
   },
   "dependencies": {
     "commander": "^3.0.2",
-    "eslint": "^6.4.0",
     "express": "^4.17.1",
```
Comment (e-e-e) [inline package.json:23]: We can also probably remove express as it is currently only used by the example `server.js`

**Review by e-e-e — APPROVED** [review]
Thank you! These contributions are great and this little tool closer towards are production ready library. If you are interested I am happy to add you as an admin for this package and we can maintain it together. If not no worries, I am happy to keep reviewing and merging any PRs.

**Comment by raineorshine** [comment]
> Thank you! These contributions are great and this little tool closer towards are production ready library. If you are interested I am happy to add you as an admin for this package and we can maintain it together. If not no worries, I am happy to keep reviewing and merging any PRs.

Sure, happy to help. Thanks for including me.

**Comment by e-e-e** [comment]
Invite sent.

**Pull request merged**